### DISCUSSION TITLE
程式可以下載並安裝在哪裡？

### DISCUSSION THREAD

**Discussion by enoch-lightning** [1101-GS3073-C/Python-Community-GS3073-C#236] [Q&A] [answered]
因為電腦常常會因為空間不夠，導致想安裝的程式無法安裝的情況。為了解決這個問題，我便突然有個想法：是否能把程式，如python、matlab、C、C++、R、FORTRAN、perl、SQL、SQLite、MySQL、PySQL、組合語言與LabVIEW等各式各樣的程式語言放在隨身碟、硬碟(包含HDD與SSD)裡面，想要開啟這些程式語言，就把下載並安裝好程式的隨身碟或硬碟，接到電腦上，便可以開始寫這些程式。今天將這樣的想法問老師，老師說不行將這些程式裝在隨身碟裡。至於硬碟(HDD與SSD)，我不知道是否可以。如果可以，一定很方便。我可以買一個硬碟，把想要寫的程式放在裡面，不佔電腦的空間。如果想要寫程式，只要將硬碟接在電腦上面即可。

**Comment by fatsoap** [comment, marked as answer]
如果用 docker 的話是不是可以解決呢? 就我所知 docker image 可以會下載成 zip 檔並放入隨身碟

**Reply by enoch-lightning** [reply]
雖然看不懂，但還是謝謝你。
